src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenThatWeHaveErrorCodes.cs

```diff
@@ -52,6 +55,22 @@ public void ThereAreNoGapsDuplicatesOrIncorrectlyFormattedCodes()
             }
         }
 
+        [Fact]
+        public void ResxIsCommentedWithCorrectStrBegin()
+        {
+            var doc = XDocument.Load("Strings.resx");
+            var ns = doc.Root.Name.Namespace;
+
+            foreach (var data in doc.Root.Elements(ns + "data"))
+            {
+                var value = data.Element(ns + "value").Value;
+                var comment = data.Element(ns + "comment")?.Value ?? "";
+                var prefix = value.Substring(0, value.IndexOf(' '));
+                
+                comment.Should().StartWith(@$"{{StrBegin=""{prefix} ""}}");
```

Comment:
Apparently `@$` is too new a feature for us :(

Reply:
Oh man, I had bumped the lang ver for other fancy stuff I didn't need and reverted the props after removing the fanciness. I did not even know this was a feature, but I hope it just tolerates either order now. I can never remember the order.

Reply:
Fixed.